### PR TITLE
[github-actions] use specific OT commit for testing

### PR DESCRIPTION
**Pull request by simonlingoogle**

### CODE REVIEW

**Comment by wgtdkp** [comment]
@simonlingoogle What is the motivation of this change?

**Comment by simonlingoogle** [comment]
> @simonlingoogle What is the motivation of this change?

Currently, OTNS depends on OT master and OT depends on OTNS master. This will prohibit any incompatible change to OT or OTNS. 

We tend to avoid dependences like this, e.g. 
- https://github.com/openthread/openthread/pull/5547 depends on a specific ot-br-posix commit rather than master. 
- https://github.com/openthread/ot-commissioner/pull/127 removes the dependency on ot-br-posix from ot-commissioner

That why I also created https://github.com/openthread/openthread/pull/5553 to remove such dependency between OTNS and OT.

File: .github/workflows/stress.yml [hunk 1]
```diff
@@ -28,6 +28,9 @@ name: Stress
 
 on: [push, pull_request]
 
+env:
+  OT_COMMIT: "0d8a99bd0573227f0751e18b584b069183d1eb3f"
```
Comment (wgtdkp) [inline .github/workflows/stress.yml:32]: nit: give a documentation for this commits, or if possible,  create a tag for it.

**Review by wgtdkp — APPROVED** [review]
LGTM 👍

File: .github/workflows/stress.yml (hunk 1 above)
Comment (simonlingoogle) [reply .github/workflows/stress.yml:32]: What about adding a date comment. I think it's infeasible to use a tag whenever the commit change.

Comment (simonlingoogle) [reply .github/workflows/stress.yml]: date added

**Review by jwhui — APPROVED** [review]

**Pull request merged**